snippets/crowdstrike-falcon-actors/crowdstrike-falcon-actors.storm

```diff
@@ -0,0 +1,116 @@
+/*
+Name: crowdstrike-falcon-actors.storm
+Original Author: bartosz.roszewski@smartcontract.com
+Modified By: <>
```

Comment:
Could probably just leave this line out if it wasn't modified

Reply:
removed